packages/medusa/src/loaders/api.ts

```diff
@@ -27,5 +31,18 @@ export default async ({ app, container, configModule }: Options) => {
   app.use(bodyParser.json())
   app.use("/", routes(container, configModule.projectConfig))
 
+  try {
+    /**
+     * Register the Medusa CORE API routes using the file based routing.
+     */
+    await new RoutesLoader({
+      app,
+      rootDir: path.join(__dirname, "../api/routes"),
```

Comment:
thought: I guess this means we will traverse all directories for API routes, right? A good side benefit of using specific paths for the RoutesLoader is that it limits the amount of crawling we need to do

Reply:
True, I can update the route loader to accept regex to shorten the crawl. The approach I pasted above needs some folder management changes that doesn't look too savory.

Reply:
I see. Maybe @kasperkristensen has some input on this?